code/modules/events/ion_storm.dm

```diff
@@ -568,5 +571,27 @@
 							message = "ALL [ionthreats] ARE NOW NAMED [ionspecies]."
 						if(4)
 							message = "ALL [ionthreats] ARE NOW NAMED [ionobjects]."
+		if(42 to 46)//Pets are something else now
+			switch(rand(1,4))//What are they now?
+				if(1)//They're a human
+					message = "[ionpet] IS HUMAN"
+				if(2)//I want this to roll multiple times and can't figure out a better way, please help)
```

Comment:
I know this is scuffed but if anyone knows a better way, please let me know

Reply:
weighted list

Reply:
1 to 3

Reply:
Okay thank you